javatests/arcs/core/host/LifecycleTest.kt

```diff
@@ -212,4 +215,14 @@ class LifecycleTest {
         val particle: ReadWriteAccessParticle = testHost.getParticle(arc.id, name)
         assertThat(particle.errors).isEmpty()
     }
+
+    @Test
+    fun pipeline() = runTest {
```

Comment:
In this test or another test: Is there some way you can trigger an arc restart in order to check that `onFirstStart` will occur once?

Reply:
allocator_restartArcInTwoExternalHosts in AllocatorTestBase.kt checks that.

Reply:
Great! Thanks